etc/Makefile.am

```diff
@@ -27,8 +28,11 @@ CLEANFILES = mptcpd.conf
 # writable.
 install-data-hook: installcheck-local
 	chmod o-w $(DESTDIR)$(pkgsysconfdir)
+	mkdir -p $(DESTDIR)$(pluginsconfdir)
```

Comment:
Use `$(MKDIR_P)` instead of `mkdir -p` to be consistent with the rules generated by Automake.